app/assets/v2/js/pages/bounty_details.js

```diff
@@ -157,6 +157,10 @@ var callbacks = {
     if (val === null) {
       return [ null, null ];
     }
+    var rates_estimate = get_rates_estimate(val);
+
+    $('#value_in_usdt_wrapper').attr('title', rates_estimate);
```

Comment:
Could we wrap it under a div to have the same effect as the other tooltips ?
` '<div class="tooltip-info tooltip-sm">' +rates_estimate + '</div>'` like [here](https://github.com/gitcoinco/web/blob/400c2ae6596320c6db1306347b450a78e422ae44/app/dashboard/templates/bounty_details.html#L192)

Reply:
good call.. updating